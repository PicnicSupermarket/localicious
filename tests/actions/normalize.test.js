const { loadYaml } = require("../../src/utils/yamlUtils");
const { normalizeYaml } = require("../../src/actions/normalize");

test("normalizing works as expected with a single collection", () => {
  const res = loadYaml("tests/input/localicipe.yaml").map((data) =>
    normalizeYaml(data, ["en", "nl"], ["COLLECTION-SHARED"])
  );
  expect(res.value).toMatchSnapshot();
});

test("normalizing works as expected with multiple collections", () => {
  const res = loadYaml("tests/input/localicipe.yaml").map((data) =>
    normalizeYaml(data, ["en", "nl"], ["COLLECTION-A", "COLLECTION-B", "COLLECTION-SHARED"])
  );
  expect(res.value).toMatchSnapshot();
});
