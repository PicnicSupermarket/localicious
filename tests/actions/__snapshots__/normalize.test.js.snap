// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`normalizing works as expected with Android only 1`] = `
Array [
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Total price: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Checkout",
      "OrderOverview",
      "Total",
    ],
    "language": "en",
    "platform": "ANDROID",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Totaal: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Checkout",
      "OrderOverview",
      "Total",
    ],
    "language": "nl",
    "platform": "ANDROID",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Total price before taxes: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Checkout",
      "OrderOverview",
      "Subtotal",
    ],
    "language": "en",
    "platform": "ANDROID",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Subtotaal: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Checkout",
      "OrderOverview",
      "Subtotal",
    ],
    "language": "nl",
    "platform": "ANDROID",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "We support nesting as deep as you like",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Something",
      "With",
      "An",
      "Arbitrary",
      "Amount",
      "Of",
      "Nesting",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Je kan zo veel lagen maken als je zelf wil",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Something",
      "With",
      "An",
      "Arbitrary",
      "Amount",
      "Of",
      "Nesting",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Help: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "Title",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Help %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "Title",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": Object {
        "ONE": "%1{{d}} Pending order",
        "OTHER": "%1{{d}} Pending Orders",
        "ZERO": "%1{{d}} Pending order",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SubTitle",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": Object {
        "ONE": "%1{{d}} Lopende bestelling",
        "OTHER": "%1{{d}} Lopende bestellingen",
        "ZERO": "%1{{d}} Lopende bestelling",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SubTitle",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This > is a special <<Character>>",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SpecialCharacters",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit > is een speciaal <<Karakter>>",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SpecialCharacters",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This is a multiline string that may contain <a href=\\"about:blank\\">links</a>
You better watch out!

It may also contain empty lines to break up text into paragraphs.",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "MultilineStrings",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit is een string van meerdere regels die <a href=\\"about:blank\\">links</a> kan bevatten.
Let maar eens op!

De tekst kan zelfs witregels bevatten!",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "MultilineStrings",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": "This is the hint",
        "type": "SINGULAR",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": "This is the label",
        "type": "SINGULAR",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This is a component with accessiblity support",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Accessible",
      "RegularComponent",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": "Dit is de hint",
        "type": "SINGULAR",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": "Dit is de label",
        "type": "SINGULAR",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit is een component met accessiblity support",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Accessible",
      "RegularComponent",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular hint",
          "OTHER": "The plural hint",
        },
        "type": "PLURAL",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular label",
          "OTHER": "The plural label",
        },
        "type": "PLURAL",
      },
      "VALUE": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular value",
          "OTHER": "The plural value",
          "ZERO": "No value for this component",
        },
        "type": "PLURAL",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": Object {
        "ONE": "The singular copy",
        "OTHER": "The plural copy",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Accessible",
      "PluralComponent",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige hint",
          "OTHER": "De meervoudige hint",
        },
        "type": "PLURAL",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige label",
          "OTHER": "De meervoudige label",
        },
        "type": "PLURAL",
      },
      "VALUE": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige waarde",
          "OTHER": "The meervoudige waarde",
          "ZERO": "Geen waarde voor dit component",
        },
        "type": "PLURAL",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": Object {
        "ONE": "De enkelvoudige copy",
        "OTHER": "De meervoudige copy",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Accessible",
      "PluralComponent",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
]
`;

exports[`normalizing works as expected with all platforms 1`] = `
Array [
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Stay up to date",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Settings",
      "PushPermissionsRequest",
      "Title",
    ],
    "language": "en",
    "platform": "IOS",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Blijf op de hoogte",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Settings",
      "PushPermissionsRequest",
      "Title",
    ],
    "language": "nl",
    "platform": "IOS",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Stay up to date",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Settings",
      "PushPermissionsRequest",
      "Subtitle",
    ],
    "language": "en",
    "platform": "IOS",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Blijf op de hoogte",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Settings",
      "PushPermissionsRequest",
      "Subtitle",
    ],
    "language": "nl",
    "platform": "IOS",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Total price: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Checkout",
      "OrderOverview",
      "Total",
    ],
    "language": "en",
    "platform": "ANDROID",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Totaal: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Checkout",
      "OrderOverview",
      "Total",
    ],
    "language": "nl",
    "platform": "ANDROID",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Total price before taxes: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Checkout",
      "OrderOverview",
      "Subtotal",
    ],
    "language": "en",
    "platform": "ANDROID",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Subtotaal: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Checkout",
      "OrderOverview",
      "Subtotal",
    ],
    "language": "nl",
    "platform": "ANDROID",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "We support nesting as deep as you like",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Something",
      "With",
      "An",
      "Arbitrary",
      "Amount",
      "Of",
      "Nesting",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Je kan zo veel lagen maken als je zelf wil",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Something",
      "With",
      "An",
      "Arbitrary",
      "Amount",
      "Of",
      "Nesting",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Help: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "Title",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Help %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "Title",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": Object {
        "ONE": "%1{{d}} Pending order",
        "OTHER": "%1{{d}} Pending Orders",
        "ZERO": "%1{{d}} Pending order",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SubTitle",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": Object {
        "ONE": "%1{{d}} Lopende bestelling",
        "OTHER": "%1{{d}} Lopende bestellingen",
        "ZERO": "%1{{d}} Lopende bestelling",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SubTitle",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This > is a special <<Character>>",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SpecialCharacters",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit > is een speciaal <<Karakter>>",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SpecialCharacters",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This is a multiline string that may contain <a href=\\"about:blank\\">links</a>
You better watch out!

It may also contain empty lines to break up text into paragraphs.",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "MultilineStrings",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit is een string van meerdere regels die <a href=\\"about:blank\\">links</a> kan bevatten.
Let maar eens op!

De tekst kan zelfs witregels bevatten!",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "MultilineStrings",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": "This is the hint",
        "type": "SINGULAR",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": "This is the label",
        "type": "SINGULAR",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This is a component with accessiblity support",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Accessible",
      "RegularComponent",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": "Dit is de hint",
        "type": "SINGULAR",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": "Dit is de label",
        "type": "SINGULAR",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit is een component met accessiblity support",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Accessible",
      "RegularComponent",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular hint",
          "OTHER": "The plural hint",
        },
        "type": "PLURAL",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular label",
          "OTHER": "The plural label",
        },
        "type": "PLURAL",
      },
      "VALUE": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular value",
          "OTHER": "The plural value",
          "ZERO": "No value for this component",
        },
        "type": "PLURAL",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": Object {
        "ONE": "The singular copy",
        "OTHER": "The plural copy",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Accessible",
      "PluralComponent",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige hint",
          "OTHER": "De meervoudige hint",
        },
        "type": "PLURAL",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige label",
          "OTHER": "De meervoudige label",
        },
        "type": "PLURAL",
      },
      "VALUE": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige waarde",
          "OTHER": "The meervoudige waarde",
          "ZERO": "Geen waarde voor dit component",
        },
        "type": "PLURAL",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": Object {
        "ONE": "De enkelvoudige copy",
        "OTHER": "De meervoudige copy",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Accessible",
      "PluralComponent",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
]
`;

exports[`normalizing works as expected with iOS only 1`] = `
Array [
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Stay up to date",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Settings",
      "PushPermissionsRequest",
      "Title",
    ],
    "language": "en",
    "platform": "IOS",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Blijf op de hoogte",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Settings",
      "PushPermissionsRequest",
      "Title",
    ],
    "language": "nl",
    "platform": "IOS",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Stay up to date",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Settings",
      "PushPermissionsRequest",
      "Subtitle",
    ],
    "language": "en",
    "platform": "IOS",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Blijf op de hoogte",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Settings",
      "PushPermissionsRequest",
      "Subtitle",
    ],
    "language": "nl",
    "platform": "IOS",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "We support nesting as deep as you like",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Something",
      "With",
      "An",
      "Arbitrary",
      "Amount",
      "Of",
      "Nesting",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Je kan zo veel lagen maken als je zelf wil",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Something",
      "With",
      "An",
      "Arbitrary",
      "Amount",
      "Of",
      "Nesting",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Help: %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "Title",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": "Help %1{{s}}",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "Title",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": Object {
        "ONE": "%1{{d}} Pending order",
        "OTHER": "%1{{d}} Pending Orders",
        "ZERO": "%1{{d}} Pending order",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SubTitle",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": true,
      "translation": Object {
        "ONE": "%1{{d}} Lopende bestelling",
        "OTHER": "%1{{d}} Lopende bestellingen",
        "ZERO": "%1{{d}} Lopende bestelling",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SubTitle",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This > is a special <<Character>>",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SpecialCharacters",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit > is een speciaal <<Karakter>>",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "SpecialCharacters",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This is a multiline string that may contain <a href=\\"about:blank\\">links</a>
You better watch out!

It may also contain empty lines to break up text into paragraphs.",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "MultilineStrings",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit is een string van meerdere regels die <a href=\\"about:blank\\">links</a> kan bevatten.
Let maar eens op!

De tekst kan zelfs witregels bevatten!",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Delivery",
      "Widget",
      "MultilineStrings",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": "This is the hint",
        "type": "SINGULAR",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": "This is the label",
        "type": "SINGULAR",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": "This is a component with accessiblity support",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Accessible",
      "RegularComponent",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": "Dit is de hint",
        "type": "SINGULAR",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": "Dit is de label",
        "type": "SINGULAR",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": "Dit is een component met accessiblity support",
      "type": "SINGULAR",
    },
    "keyPath": Array [
      "Accessible",
      "RegularComponent",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular hint",
          "OTHER": "The plural hint",
        },
        "type": "PLURAL",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular label",
          "OTHER": "The plural label",
        },
        "type": "PLURAL",
      },
      "VALUE": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "The singular value",
          "OTHER": "The plural value",
          "ZERO": "No value for this component",
        },
        "type": "PLURAL",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": Object {
        "ONE": "The singular copy",
        "OTHER": "The plural copy",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Accessible",
      "PluralComponent",
    ],
    "language": "en",
    "platform": "SHARED",
  },
  Object {
    "ACCESSIBILITY": Object {
      "HINT": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige hint",
          "OTHER": "De meervoudige hint",
        },
        "type": "PLURAL",
      },
      "LABEL": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige label",
          "OTHER": "De meervoudige label",
        },
        "type": "PLURAL",
      },
      "VALUE": Object {
        "containsFormatting": false,
        "translation": Object {
          "ONE": "De enkelvoudige waarde",
          "OTHER": "The meervoudige waarde",
          "ZERO": "Geen waarde voor dit component",
        },
        "type": "PLURAL",
      },
    },
    "COPY": Object {
      "containsFormatting": false,
      "translation": Object {
        "ONE": "De enkelvoudige copy",
        "OTHER": "De meervoudige copy",
      },
      "type": "PLURAL",
    },
    "keyPath": Array [
      "Accessible",
      "PluralComponent",
    ],
    "language": "nl",
    "platform": "SHARED",
  },
]
`;
